tests/test_processors.py

```diff
@@ -253,7 +253,7 @@ def build_event(headers):
         return flag_robots(event)
 
     assert build_event(request_headers["user"])["is_robot"] is False
-    assert build_event(request_headers["machine"])["is_robot"] is False
+    assert build_event(request_headers["machine"])["is_robot"] is True
```

Comment:
I think this should be commented out instead. Maybe we fix in https://github.com/inveniosoftware/invenio-stats/pull/154 and merge that first?